examples/experiments/stlc_strategies.makam

```diff
@@ -97,3 +99,8 @@ stlc_strategies : testsuite.
 >> CBND_VAL := tuple [  ],
 >> CBND_COUNTER := 0.
 
+>> (eq _TERM (app (lam (fun x => x)) side_effect),
```

Comment:
I'm not completely convinced this example/test makes sense, but it was the checker for this issue.

Reply:
I think it's a good example for this issue!